### DISCUSSION TITLE
Receiving multicast on docker

### DISCUSSION THREAD

**Discussion by avargasn** [troglobit/mcjoin#30] [Q&A] [answered]
Have you successfully received the multicast stream on docker? Is there anything to do to allow it? I'm running on EVE-NG.

**Comment by troglobit** [comment, marked as answer]
I have no idea how EVE-NG works, but multicast is a bit tricky, as you may know already.  When I run `mcjoin -s` on another PC as sender, I can start a receiver in docker as follows on my laptop:

    docker run -it --network="host" troglobit/mcjoin

Otherwise you need to have a routing daemon that can route-in the multicast mcjoin requests.

**Comment by avargasn** [comment]
Joachim,

Thank you very much for your kind reply.

I will investigate this issue further.

Many thanks,

Alberto

Em dom., 15 de jan. de 2023 às 02:25, Joachim Wiberg <
***@***.***> escreveu:
